src/components/Calculator.js

```diff
@@ -1,34 +1,93 @@
 import React, { Component } from 'react';
+import calculate from './logic/calculate';
+import './Calculator.css';
 
 // eslint-disable-next-line react/prefer-stateless-function
 class Calculator extends Component {
+  constructor(props) {
+    super(props);
```

Comment:
- [ ] Great job so far, it is better to solve the error than disable the linter, kindly fix the errors, if any.